backend/btrixcloud/users.py

```diff
@@ -488,6 +489,11 @@ async def delete_invite(token: str):
         await user_manager.invites.remove_invite(token)
         return {"removed": True}
 
+    @users_router.get("/invites", tags=["invites"])
```

Comment:
I think for the specific issue, this should probably be `/api/org/<oid>/invites`, and scope the invites only to a specific org. That should have owner permission.

We could still have this endpoint too (viewing all invites), but should probably only allow superadmin access to this?

Reply:
For now, lets:
- Add superuser check to this endpoint
- Add /api/orgs/<oid>/invites with org owner check